### PR TITLE
Dls update flow

### PR DESCRIPTION
**Pull request by aydin101**

### CODE REVIEW

File: ui_src/src/domain/stationOverview/components/useSchemaModal/index.js [hunk 1]
```diff
@@ -28,26 +28,35 @@ import Modal from '../../../../components/modal';
 import pathDomains from '../../../../router';
 import { StationStoreContext } from '../..';
 import SchemaItem from './schemaItem';
+import stationImg from '../../../../assets/images/stationsIconActive.svg';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/components/useSchemaModal/index.js:31]: Why you didn't import as you suggested?

File: ui_src/src/domain/stationOverview/stationObservabilty/messages/index.js [hunk 2]
```diff
@@ -51,6 +56,12 @@ const Messages = () => {
     const [subTabValue, setSubTabValue] = useState('Unacked');
     const [tabValue, setTabValue] = useState('Messages');
     const [isCheck, setIsCheck] = useState([]);
+    // const [isDls, setIsDls] = useState(false);
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationOverview/stationObservabilty/messages/index.js:59]: remove comment

**Comment by avrhamNeeman** [comment]
image
The dot is unnecessary here.

**Comment by avrhamNeeman** [comment]
image
The station names are not the same.

**Comment by avrhamNeeman** [comment]
When I create a new station from the consumption modal, the list is not updated.
image

**Comment by avrhamNeeman** [comment]
image
"No stations yet"
"Get started by creating your station"

**Comment by avrhamNeeman** [comment]
When creating a new station from the consumption modal, it should update the list without redirecting to the new station.

**Comment by avrhamNeeman** [comment]
image
-> Search station

**Review by avrhamNeeman — APPROVED** [review]

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**